src/components/content/Content.module.sass

```diff
@@ -11,10 +11,27 @@
 
         main
             padding-block: 2rem
-            max-width: calc( 100% - 16rem )
```

Comment:
This is the base design for desktop. Can be changed through media queries, but not removed.